packages/storage-plugin/tests/storage.plugin.spec.ts

```diff
@@ -196,13 +195,13 @@ describe('NgxsStoragePlugin', () => {
 
     // Assert
     expect(localStorage.getItem('@@STATE')).toBe(
-      JSON.stringify({ counter: { counts: 100, version: 2 } })
+      JSON.stringify({ counter: { counts: 100, version: 1 } })
     );
   });
 
   it('should migrate single localstorage', () => {
     // Arrange
-    const data = JSON.stringify({ count: 100, version: 1 });
```

Comment:
Do you have to change these tests? Can't add a new one?

Reply:
This is because of the breaking change.

Reply:
Unfortunately, we cannot allow critical changes.

Reply:
In bug fixes, or also for feature requests?

Reply:
@kayvanbree The semantic versioning assumes that breaking changes must be published with major release. But we can't easily switch to v4.

Reply:
Yeah I get that. Are there any plans or breaking changes scheduled for v4?

Reply:
https://github.com/ngxs/store/issues/827